packages/components/src/date-time/date/index.tsx

```diff
@@ -15,12 +16,14 @@ import DayPickerSingleDateController from 'react-dates/lib/components/DayPickerS
  */
 import { useEffect, useRef } from '@wordpress/element';
 import { isRTL, _n, sprintf } from '@wordpress/i18n';
+import { arrowLeft, arrowRight } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import { getMomentDate } from './utils';
-import type { DatePickerDayProps, DatePickerProps } from './types';
+import type { DatePickerDayProps, DatePickerProps } from '../types';
+import { Day, NavPrevButton, NavNextButton } from './styles';
 
 const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
 const ARIAL_LABEL_TIME_FORMAT = 'dddd, LL';
```

Comment:
Not necessary for this PR (it's in trunk) I noticed that the aria-label on the day component outputs symbols.

<img width="1119" alt="Screen Shot 2022-05-27 at 11 17 15 am" src="https://user-images.githubusercontent.com/6458278/170611316-f0f499da-9e82-4916-8589-c285781077c7.png">


Not sure if the problem is in `moment()`. It's something to do with the locale aware format  `LL` 

This works

```js
const ARIAL_LABEL_TIME_FORMAT = 'dddd, MMMM DD, YYYY';
```

But it's not locale aware I presume so it might need some further 👀

Reply:
Oh interesting, good catch. We probably should use our own `dateFormat` function that's in `@wordpress/date` here:

https://github.com/WordPress/gutenberg/blob/ef6c2fe106788c7efec627606b3de892974a0d9d/packages/components/src/date-time/date.tsx#L44

I'll turn your comment into an issue.

Reply:
https://github.com/WordPress/gutenberg/issues/41384